src/index.ts

```diff
@@ -52,8 +53,9 @@ export default function mitt(all: EventHandlerMap): Emitter {
 		 * @memberOf mitt
 		 */
 		off(type: EventType, handler: Handler) {
-			if (all.has(type)) {
-				all.get(type).splice(all.get(type).indexOf(handler) >>> 0, 1);
+			const handlers = all.get(type);
+			if (handlers) {
+				handlers.splice(handlers.indexOf(handler) >>> 0, 1);
```

Comment:
Wouldn't it be better to use `Set` for the handlers instead?

Reply:
For speed yes, but Set rejects duplicates. EventEmitter, the API contract mitt adheres to, allows duplicate handlers.

Reply:
Does that mean I can register the same functions twice? That is really interesting.